modules/telemetry/include/hephaestus/telemetry/measure_sink.h

```diff
@@ -0,0 +1,29 @@
+//=================================================================================================
+// Copyright (C) 2023-2024 HEPHAESTUS Contributors
+//=================================================================================================
+
+#pragma once
+
+#include <chrono>
+#include <string>
+
+namespace heph::telemetry {
+using ClockT = std::chrono::system_clock;
+
+struct MeasureEntry {
+  [[nodiscard]] auto operator==(const MeasureEntry& other) const -> bool = default;
+
+  std::string component;
+  std::string tag;
```

Comment:
some explanatory comment would be cool here since component and tag are not self explanatory names.